playground/src/modules/x/counter/counter.html

```diff
@@ -1,5 +1,5 @@
 <template>
-    Counter: {counter}
-    <button onclick={decrement}>-</button>
-    <button onclick={increment}>+</button>
-</template>
+  Counter: {counter}
+  <button onclick={decrement}>-</button>
+  <button onclick={increment}>+</button>
+</template>
```

Comment:
This change seems unrelated to this PR?

Reply:
This was accidental, there is supposed to be no change here